fossevents/pages/templates/pages/event_detail.html

```diff
@@ -1,22 +1,33 @@
 {% extends "base.html" %}
 {% load django_markdown %}
-{% block body_classes %}{{block.super}}page--event-detail{% endblock body_classes %}
-{% block title %}{{ event.name }} • {{block.super}}{% endblock title %}
```

Comment:
There two lines are responsible for adding `<title>` tag and a top level `class` attribute to `<body>` tag. What is the clash that you are getting?